### PR TITLE
Rename JUCE.cmake to FRUT

### PR DESCRIPTION
**Pull request by McMartin**
That's a much cooler name!

@MartyLake: FYI

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 [![Build status][appveyor-badge]][appveyor]
 [![Build Status][travis-ci-badge]][travis-ci]
-[![Join the chat at https://gitter.im/JUCE-cmake/Lobby][gitter-badge]][gitter]
+[![Join the chat at https://gitter.im/FRUT/Lobby][gitter-badge]][gitter]
```
Comment (MartyLake) [inline README.md]: frut lobby is not registered

Comment (McMartin) [reply README.md]: I know. I intend to remove it in a subsequent PR, since it's unused.

**Comment by MartyLake** [comment]
Lgtm then ^_^

Le 18 oct. 2017 11:44 PM, "Alain Martin" <notifications@github.com> a
écrit :

**Comment by McMartin** [comment]
I decided to do the cleanup on this PR in the end.

**Pull request merged**